<?xml version="1.0"?>
<launch>
	
	<!-- send youbot urdf to param server -->
	<param name="robot_description" command="$(find xacro)/xacro.py '$(find youbot_description)/robots/youbot.urdf.xacro'"/>
        <!--<remap from="joint_states" to="youbot_state"/>-->

	<!-- start robot_state_publisher -->
  	<node pkg="robot_state_publisher" type="state_publisher" name="rob_st_pub" />
	
</launch>
